### PR TITLE
OSDOCS3023: Updating clusters overview enhancement

### PR DESCRIPTION
**Pull request by xenolinux**
This PR addresses the enhancement in the updating clusters overview page as per the suggestions in this PR: https://github.com/https://github.com/openshift/openshift-docs/pull/45298

Version(s): 4.8+

Issue: [Jira](https://issues.redhat.com/browse/OSDOCS-3023)

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/updating-clusters-overview-enhancement/updating/index.html

Additional information:

**QE** ack is not required

### CODE REVIEW

File: updating/index.adoc [hunk 1]
```diff
@@ -83,7 +83,7 @@ xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-ne
 [id="updating-clusters-overview-vsphere-updating-hardware"]
 == Updating hardware on nodes running on vSphere
```
Comment (Amrita42) [inline updating/index.adoc:84]: I think it should be **running in vSphere**

File: updating/index.adoc [hunk 2]
```diff
@@ -24,13 +24,13 @@ xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgra
...
 
 * xref:../updating/preparing-eus-eus-upgrade.adoc#updating-eus-to-eus-upgrade_eus-to-eus-upgrade[Updating EUS-to-EUS]
 
 [id="updating-clusters-overview-update-cluster-using-web-console"]
 == Updating a cluster using the web console
-xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster within a minor version using the web console]: You can update an {product-title} cluster by using the web console. You can the update a cluster between minor versions. For more information, see the following items:
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster using the web console]: You can update an {product-title} cluster by using the web console. The following steps update a cluster within a minor version. You can use the same instructions for updating a cluster between minor versions.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply updating/index.adoc:33]: hmm, there had been a discussion about these wordings in the [original PR](https://github.com/openshift/openshift-docs/pull/45298).  To maintain the consistency between [updating the cluster using the CLI ](https://docs.openshift.com/container-platform/4.10/updating/updating-cluster-cli.html)and the [web console ](https://docs.openshift.com/container-platform/4.10/updating/updating-cluster-within-minor.html)sections.  The same lines have been kept on an overview page. 

But I can open a follow-up PR for the main sections to change those two lines and make them one. Let me know your thoughts.

File: updating/index.adoc [hunk 3]
```diff
@@ -49,7 +49,7 @@ xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a clust
...
+xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools[Performing a canary rollout update]: By controlling rollout of an update to the worker nodes, you can ensure that mission-critical applications stay available during the whole update, even if the update process causes your applications to fail. Depending on your organizational needs, you might want to update a small subset of worker nodes, evaluate cluster and workload health over a period of time, and then update the remaining nodes. This is referred to as a _canary_ update. Alternatively, you might also want to fit worker node updates, which often requires a host reboot, into smaller defined maintenance windows when it is not possible to take a large maintenance window to update the entire cluster at one time. You can perform the following procedures:
```
Comment (michaelryanpeter) [inline updating/index.adoc:52]:
```suggestion
xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools[Performing a canary rollout update]: By controlling the rollout of an update to the worker nodes, you can ensure that mission-critical applications stay available during the whole update, even if the update process causes your applications to fail. Depending on your organizational needs, you might want to update a small subset of worker nodes, evaluate cluster and workload health over a period of time, and then update the remaining nodes. This is referred to as a _canary_ update. Alternatively, you might also want to fit worker node updates, which often requires a host reboot, into smaller defined maintenance windows when it is not possible to take a large maintenance window to update the entire cluster at one time. You can perform the following procedures:
```

File: updating/index.adoc [hunk 4]
```diff
@@ -59,15 +59,15 @@ xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-cust
...
 
 [id="updating-clusters-overview-update-restricted-network-cluster"]
 == Updating a restricted network cluster
-xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: A restricted network environment is one in which your cluster nodes cannot access the internet. You can update a restricted network {product-title} cluster by using the `oc`. You can also update a restricted network {product-title} cluster by using the OpenShift Update Service. For more information, see the following items:
+xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: If your mirror host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment and then bring that host or removable media across that gap. If the local container registry and the cluster are connected to the mirror host of a registry, you can directly push the release images to the local registry.
```
Comment (michaelryanpeter) [inline updating/index.adoc:70]:
```suggestion
xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: If your mirror host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment and then bring that host or removable media across that gap. If the local container registry and the cluster are connected to the mirror host of a registry, you can directly push the release images to the local registry.
```
This first sentence is very long and a little hard to follow. Could you try breaking it up into several shorter sentences?

**Review by michaelryanpeter — COMMENTED** [review]
Please let me know if you have questions!

File: updating/index.adoc (hunk 3 above)
Comment (xenolinux) [reply updating/index.adoc:52]: Addressed

File: updating/index.adoc [hunk 5]
```diff
@@ -6,15 +6,15 @@ include::_attributes/common-attributes.adoc[]
...
 == Understanding upgrade channels and releases
-xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. For more information, see the following:
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of the {product-title} always installs that minor version. For more information, see the following:
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply updating/index.adoc:17]: Addressed

File: updating/index.adoc (hunk 4 above)
Comment (xenolinux) [reply updating/index.adoc]: Addressed
